libloki/crypto.js

```diff
@@ -158,6 +158,27 @@
     }
   }
 
+  async function decryptToken(ivAndCipherText64, serverPubKey64) {
```

Comment:
Mmh I wished I had better modularised this file, since it looks like what `decryptToken` does is quite generic and is also exactly what's in `FallBackSessionCipher.decrypt(ivAndCipherText)`, but it would be weird to use "FallbBackSessionCipher".
The generic functionality of encrypting/decrypting using the user keypair should be available as private functions, probably worth a refactor later.

For now maybe replace lines 165, 166 and 177 with a call to `DHDecrypt`?